controller/pkg/packet/constants.go

```diff
@@ -111,14 +110,16 @@ const (
 
 // UDP related constants.
 const (
+	// udpSourcePortPos is the location of source port
+	udpSourcePortPos = 0
+	// udpDestPortPos is the location of destination port
+	udpDestPortPos = 2
 	// UDPLengthPos is the location of UDP length
-	UDPLengthPos = 24
+	UDPLengthPos = 4
 	// UDPChecksumPos is the location of UDP checksum
-	UDPChecksumPos = 26
+	UDPChecksumPos = 6
 	// UDPDataPos is the location of UDP data
-	UDPDataPos = 28
-	// UDPBeginPos is the location of UDP Header
-	UDPBeginPos = 20
+	UDPDataPos = 8
```

Comment:
Same for those. Do they need be public? Callers should not need them.

Reply:
datapath is using them.